plugins/@grouparoo/spec-helper/src/lib/cliSpecHelper.ts

```diff
@@ -68,7 +71,10 @@ export namespace CLISpecHelper {
   ): { exitCode: number; stderr: string; stdout: string } {
     const syncResponse = spawnSync(command, args, {
       cwd,
-      env,
+      env: {
+        ...env,
+        GROUPAROO_TELEMETRY_ENABLED: process.env.GROUPAROO_TELEMETRY_ENABLED,
```

Comment:
Maybe just always "false" bc this is always in the test context? oR is there a reason for it to be true?

Reply:
probably not in this test!